exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -106,7 +106,13 @@
   <p>STATE: ${model.getProfile().getState().name()}</p>
   <p>FOREMAN: ${model.getProfile().getForeman().getAddress()}</p>
   <p>TOTAL FRAGMENTS: ${model.getProfile().getTotalFragments()}</p>
-
+  <p>
+  <#if (model.getProfile().getState().name() == "RUNNING")> 
+    ELAPSED: ${(.now?long - model.getProfile().getStart()) / 1000} 
```

Comment:
Rather than encoding this in the template three times, perhaps compute it ahead of time and store the computed result in the "model". Makes it easier to handle the multi-unit formatting, digits of precision, etc.